rust_src/src/buffers.rs

```diff
@@ -360,20 +360,18 @@ fn set_buffer(buffer_or_name: LispObject) -> LispObject {
     buffer
 }
 
-pub fn rust_init_syms() {
-    unsafe {
-        defsubr(&*Sbuffer_chars_modified_tick);
-        defsubr(&*Sbuffer_file_name);
-        defsubr(&*Sbuffer_live_p);
-        defsubr(&*Sbuffer_modified_p);
-        defsubr(&*Sbuffer_modified_tick);
-        defsubr(&*Sbuffer_name);
-        defsubr(&*Scurrent_buffer);
-        defsubr(&*Sget_buffer);
-        defsubr(&*Soverlay_buffer);
-        defsubr(&*Soverlay_end);
-        defsubr(&*Soverlay_start);
-        defsubr(&*Soverlayp);
-        defsubr(&*Sset_buffer);
-    }
+export_lisp_fns! {
+    buffer_chars_modified_tick,
+    buffer_file_name,
+    buffer_live_p,
+    buffer_modified_p,
+    buffer_modified_tick,
+    buffer_name,
+    current_buffer,
+    get_buffer,
+    overlay_buffer,
+    overlay_end,
+    overlay_start,
+    overlayp,
+    set_buffer
```

Comment:
please put a trailing comma in those lists.

Reply:
It does not compile when I do.